assets/src/hooks/useDetour.ts

```diff
@@ -26,29 +26,10 @@ export const useDetour = (input: UseDetourInput) => {
 
   // Record snapshots when changed
   useEffect(() => {
-    const snapshotSubscription = actorRef.subscribe((snap) => {
+    const snapshotSubscription = actorRef.subscribe(() => {
       const persistedSnapshot = actorRef.getPersistedSnapshot()
       const serializedSnapshot = JSON.stringify(persistedSnapshot)
       localStorage.setItem("snapshot", serializedSnapshot)
-
-      // check for no-save tag before
-      if (snap.hasTag("no-save")) {
-        return
-      }
-
-      actorRef.getSnapshot().can({ type: "detour.save.begin-save" }) &&
-        actorRef.send({ type: "detour.save.begin-save" })
-
-      putDetourUpdate(persistedSnapshot).then((uuid) => {
-        if (
-          isOk(uuid) &&
-          actorRef
-            .getSnapshot()
-            .can({ type: "detour.save.set-uuid", uuid: uuid.ok })
-        ) {
-          actorRef.send({ type: "detour.save.set-uuid", uuid: uuid.ok })
-        }
-      })
     })
 
     return () => {
```

Comment:
Question for everyone: I think the `localStorage` save is not used anymore. That was our first stage of setting up a sort of autosave feature, so if Skate freezes up and needs a refresh, the detour in progress is retained, even before we had the db. Now I don't think it's necessary, right? In which case this whole hook could go?

Reply:
Yeah I'd remove the entire hook.